ext/sockets/conversions.c

```diff
@@ -241,26 +241,26 @@ static void from_zval_write_aggregation(const zval *container,
 	zval					*elem;
 
 	if (Z_TYPE_P(container) != IS_ARRAY) {
-		do_from_zval_err(ctx, "%s", "expected an array here");
+		zend_type_error("Expected array");
+		return;
 	}
 
 	for (descr = descriptors; descr->name != NULL && !ctx->err.has_error; descr++) {
 		if ((elem = zend_hash_str_find(Z_ARRVAL_P(container),
 				descr->name, descr->name_size - 1)) != NULL) {
 
 			if (descr->from_zval == NULL) {
```

Comment:
Convert this to `ZEND_ASSERT(descr->from_zval)`, doesn't seem like it can be hit.

Reply:
It looks like you converted the wrong condition to an assert here...

Reply:
Uhhhh, what was I thinking >_>